bin/index.js

```diff
@@ -8,25 +8,39 @@ const promisifyAll = require('bluebird').promisifyAll;
 const makeDir = promisify(require('mkdirp'));
 const fs = promisifyAll(require('fs'));
 
+function range(val) {
```

Comment:
maybe better if returns object with `min` and `max`? makes reading usage after this easier as you'll see `range.min` etc.